frontend/src/components/shared/ArchivedCommentsBox.tsx

```diff
@@ -141,12 +141,16 @@ export default function ArchivedCommentsBox({ comments, handleAddComment }: Comm
                 title={
                     <div>
                         {currentComment &&
-                            <Space>
-                                <Text key={0} strong>Added at:</Text>
-                                {dateToString(new Date(currentComment.updated_at), "compact")}
-                                <Text key={1} strong>By:</Text>
-                                {usersByID[currentComment.created_by]?.username}
-                            </Space>
+                            <Flex justify={"space-between"}>
+                                <Space direction={"horizontal"}>
+                                    <Text strong>By:</Text>
```

Comment:
I prefer the time first then the username ... You prefer the other way around ?

Reply:
I don't have a strong opinion so I can cave into yours ;)